FairRoot installation fails in environment jun19

Installation of FairRoot fails with the following error message.

```
==> Installing fairroot
==> Searching for binary cache of fairroot
==> Warning: No Spack mirrors are currently configured
==> No binary for fairroot found: installing from source
==> Warning: microarchitecture specific optimizations are not supported yet on mixed compiler toolchains [check clang@10.0.1-apple for further details]
==> Using cached archive: /Users/tstockmanns/.spack/source_cache/_source-cache/archive/06/06a5b3b2c5445f7342464061cccbe7bc.tar.gz
==> Staging archive: /var/folders/_1/1x665w457cgdjljzc_t7_l5m0000gn/T/tstockmanns/spack-stage/spack-stage-fairroot-18.2.1-urepgs4yvqfwuhuvqz6ppdhfd44iiueh/v18.2.1.tar.gz
==> Created stage in /var/folders/_1/1x665w457cgdjljzc_t7_l5m0000gn/T/tstockmanns/spack-stage/spack-stage-fairroot-18.2.1-urepgs4yvqfwuhuvqz6ppdhfd44iiueh
==> Applied patch /Users/tstockmanns/PandaSoftware/FairSoft/dev/source/packages/fairroot/cmake_utf8.patch
==> Applied patch /Users/tstockmanns/PandaSoftware/FairSoft/dev/source/packages/fairroot/fairlogger_incdir.patch
==> Building fairroot [CMakePackage]
==> Executing phase: 'cmake'
==> Error: ProcessError: Command exited with status 1:
    'cmake' '/var/folders/_1/1x665w457cgdjljzc_t7_l5m0000gn/T/tstockmanns/spack-stage/spack-stage-fairroot-18.2.1-urepgs4yvqfwuhuvqz6ppdhfd44iiueh/spack-src' '-G' 'Unix Makefiles' '-DCMAKE_INSTALL_PREFIX:PATH=/Users/tstockmanns/.spack/install_tree/darwin-mojave-x86_64/clang-10.0.1-apple/fairroot-18.2.1-urepgs4yvqfwuhuvqz6ppdhfd44iiueh' '-DCMAKE_BUILD_TYPE:STRING=RelWithDebInfo' '-DCMAKE_VERBOSE_MAKEFILE:BOOL=ON' '-DCMAKE_FIND_FRAMEWORK:STRING=LAST' '-DCMAKE_FIND_APPBUNDLE:STRING=LAST' '-DCMAKE_INSTALL_RPATH_USE_LINK_PATH:BOOL=FALSE' '-DCMAKE_INSTALL_RPATH:STRING=/Users/tstockmanns/.spack/install_tree/darwin-mojave-x86_64/clang-10.0.1-apple/fairroot-18.2.1-urepgs4yvqfwuhuvqz6ppdhfd44iiueh/lib;/Users/tstockmanns/.spack/install_tree/darwin-mojave-x86_64/clang-10.0.1-apple/fairroot-18.2.1-urepgs4yvqfwuhuvqz6ppdhfd44iiueh/lib64;/Users/tstockmanns/.spack/install_tree/darwin-mojave-x86_64/clang-10.0.1-apple/boost-1.68.0-3xkegvqksficzlnriyd7s4ikss2vp6hq/lib;/Users/tstockmanns/.spack/install_tree/darwin-mojave-x86_64/clang-10.0.1-apple/bzip2-1.0.8-llnc6sqphf7yp4drig6njzzkxctjwfse/lib;/Users/tstockmanns/.spack/install_tree/darwin-mojave-x86_64/clang-10.0.1-apple/zlib-1.2.11-nqnlnxddpa4qfz5cnf4grnct57ds25ks/lib;/Users/tstockmanns/.spack/install_tree/darwin-mojave-x86_64/clang-10.0.1-apple/fairlogger-1.4.0-4zg2pknqx2ixkof4cxdvmdyoyeurgzv6/lib;/Users/tstockmanns/.spack/install_tree/darwin-mojave-x86_64/clang-10.0.1-apple/fairmq-1.4.3-hyprxzrxh33mjjc6fveohengs7jy5lnx/lib;/Users/tstockmanns/.spack/install_tree/darwin-mojave-x86_64/clang-10.0.1-apple/dds-2.4-ex2o5wuprpvg6h2qkjpfjqgpvtwefson/lib;/Users/tstockmanns/.spack/install_tree/darwin-mojave-x86_64/clang-10.0.1-apple/googletest-1.8.1-sdcowbfxteo6x2mnlzbuh3yobr6znwzf/lib;/Users/tstockmanns/.spack/install_tree/darwin-mojave-x86_64/clang-10.0.1-apple/msgpack-c-3.1.1-4yysnq2n4hpxp56ptravssic7jufpz3b/lib;/Users/tstockmanns/.spack/install_tree/darwin-mojave-x86_64/clang-10.0.1-apple/nanomsg-1.1.5-s72gp5m4w6ipesib6qbvjdwmgny3pmbk/lib;/Users/tstockmanns/.spack/install_tree/darwin-mojave-x86_64/clang-10.0.1-apple/pmix-3.1.3-l7tmz4ym67tkv4qhmryha3pwoiwopbyq/lib;/Users/tstockmanns/.spack/install_tree/darwin-mojave-x86_64/clang-10.0.1-apple/hwloc-2.0.2-evou2fftetteiou6ohykcbv5hslx6n2t/lib;/Users/tstockmanns/.spack/install_tree/darwin-mojave-x86_64/clang-10.0.1-apple/libxml2-2.9.9-rfcckf4qb62ko44mqt3xu2yg33h5o5qr/lib;/Users/tstockmanns/.spack/install_tree/darwin-mojave-x86_64/clang-10.0.1-apple/libiconv-1.16-aj4spetztxsiscso7zrnkrhqlp2r4ln7/lib;/Users/tstockmanns/.spack/install_tree/darwin-mojave-x86_64/clang-10.0.1-apple/xz-5.2.4-snwzcouby3x3ztkutupqzmqwbbxh2tur/lib;/Users/tstockmanns/.spack/install_tree/darwin-mojave-x86_64/clang-10.0.1-apple/libevent-2.1.8-zsh7k4t5kdz2tngoexyetsssvpmqzvoz/lib;/Users/tstockmanns/.spack/install_tree/darwin-mojave-x86_64/clang-10.0.1-apple/openssl-1.1.1g-finquttzhrvdqss3nyo7lrmmdu3j67mf/lib;/Users/tstockmanns/.spack/install_tree/darwin-mojave-x86_64/clang-10.0.1-apple/zeromq-4.3.2-2ixpwtbng5gfjfawubl35pgqjniircjk/lib;/Users/tstockmanns/.spack/install_tree/darwin-mojave-x86_64/clang-10.0.1-apple/libsodium-1.0.17-mz2mdavum5svupz2bx7du2mnzfcwixsj/lib;/Users/tstockmanns/.spack/install_tree/darwin-mojave-x86_64/clang-10.0.1-apple/flatbuffers-1.11.0-3z6x5fbnnnwrfg4kapi25jewahtrsnjp/lib;/Users/tstockmanns/.spack/install_tree/darwin-mojave-x86_64/clang-10.0.1-apple/geant3-v2-7_fairsoft-yedc7jwoscuhn5eih72e3apnr26cye7v/lib;/Users/tstockmanns/.spack/install_tree/darwin-mojave-x86_64/clang-10.0.1-apple/root-6.16.00-ejfra4res6lpxita2fjwqq5pzz3jzaxu/lib;/Users/tstockmanns/.spack/install_tree/darwin-mojave-x86_64/clang-10.0.1-apple/davix-0.6.8-4jqpt4tbrgqiw2lneds23uth4spzpv66/lib;/Users/tstockmanns/.spack/install_tree/darwin-mojave-x86_64/clang-10.0.1-apple/libuuid-1.0.3-sky4nuvihdgnn3akzm6yj6a55f2w5z3k/lib;/Users/tstockmanns/.spack/install_tree/darwin-mojave-x86_64/clang-10.0.1-apple/fontconfig-2.12.3-zctsdtq36xshizu7bre6a4xnn72l2wh6/lib;/Users/tstockmanns/.spack/install_tree/darwin-mojave-x86_64/clang-10.0.1-apple/font-util-1.3.2-ffs3rit6djw42lb4bdasnlaijqw6j4c3/lib;/Users/tstockmanns/.spack/install_tree/darwin-mojave-x86_64/clang-10.0.1-apple/freetype-2.10.1-lbou655p4o7vbapg22zcaf4w433mjlkh/lib;/Users/tstockmanns/.spack/install_tree/darwin-mojave-x86_64/clang-10.0.1-apple/libpng-1.6.37-kg64auicvgyutefasq6lsv6jt54pkfr5/lib;/Users/tstockmanns/.spack/install_tree/darwin-mojave-x86_64/clang-10.0.1-apple/gsl-2.5-qeaufn2qqrohifqcoan2tigxwvfll5vt/lib;/Users/tstockmanns/.spack/install_tree/darwin-mojave-x86_64/clang-10.0.1-apple/intel-tbb-2019.4-v5imov3kva6qs7oqc62sxhnv3sfo7bnp/lib;/Users/tstockmanns/.spack/install_tree/darwin-mojave-x86_64/clang-10.0.1-apple/libice-1.0.9-iurcy5iq64qph4kujfwerjysbz3yo3ea/lib;/Users/tstockmanns/.spack/install_tree/darwin-mojave-x86_64/clang-10.0.1-apple/libjpeg-turbo-2.0.3-wjw25ca2ocuuqsn6f25gflkezuasj3j6/lib;/Users/tstockmanns/.spack/install_tree/darwin-mojave-x86_64/clang-10.0.1-apple/libsm-1.2.2-u3otmbm2siiplv7rgh3yonsipaxpx22m/lib;/Users/tstockmanns/.spack/install_tree/darwin-mojave-x86_64/clang-10.0.1-apple/libx11-1.6.7-nehamf2ftymjx4zwpb7wb4gu3c54vmcp/lib;/Users/tstockmanns/.spack/install_tree/darwin-mojave-x86_64/clang-10.0.1-apple/kbproto-1.0.7-cotie4iciyhayrluydxil62pe2nr7nxg/lib;/Users/tstockmanns/.spack/install_tree/darwin-mojave-x86_64/clang-10.0.1-apple/libxcb-1.13-qt2bcsdulhxc6exi7i6wl53tyq6vcstw/lib;/Users/tstockmanns/.spack/install_tree/darwin-mojave-x86_64/clang-10.0.1-apple/libpthread-stubs-0.4-koiiw7umtux5dpwr67m73ieeufwwjerr/lib;/Users/tstockmanns/.spack/install_tree/darwin-mojave-x86_64/clang-10.0.1-apple/libxau-1.0.8-jhpqtyig6tl44lsywzzhbsag3nicpp22/lib;/Users/tstockmanns/.spack/install_tree/darwin-mojave-x86_64/clang-10.0.1-apple/xproto-7.0.31-rsdzozkaq22sk6tfkloboajaysj3azzj/lib;/Users/tstockmanns/.spack/install_tree/darwin-mojave-x86_64/clang-10.0.1-apple/libxdmcp-1.1.2-c4toqy45aa5uv7z4nrxoz67rlo5jmhno/lib;/Users/tstockmanns/.spack/install_tree/darwin-mojave-x86_64/clang-10.0.1-apple/xextproto-7.3.0-pg3n3slg4ncstgcs2uzvlz34di57n7z7/lib;/Users/tstockmanns/.spack/install_tree/darwin-mojave-x86_64/clang-10.0.1-apple/libxext-1.3.3-v6cta5zm3zrdtwutbza76pirz4ky3hqk/lib;/Users/tstockmanns/.spack/install_tree/darwin-mojave-x86_64/clang-10.0.1-apple/libxft-2.3.2-hyesfp6lk6f5yjfl34nebe32siqrdq66/lib;/Users/tstockmanns/.spack/install_tree/darwin-mojave-x86_64/clang-10.0.1-apple/libxrender-0.9.10-xmnzbzujlnltwsaespjne7p7fktwvial/lib;/Users/tstockmanns/.spack/install_tree/darwin-mojave-x86_64/clang-10.0.1-apple/renderproto-0.11.1-2z7bjm673mcmnqx2cfjq6wn2m2otz6p2/lib;/Users/tstockmanns/.spack/install_tree/darwin-mojave-x86_64/clang-10.0.1-apple/libxpm-3.5.12-jkokce6effagy77ykcvpnvsq5ddpitf5/lib;/Users/tstockmanns/.spack/install_tree/darwin-mojave-x86_64/clang-10.0.1-apple/gettext-0.20.1-nm5qmlco3h37nieyp2kzkkyn5cd4hhpd/lib;/Users/tstockmanns/.spack/install_tree/darwin-mojave-x86_64/clang-10.0.1-apple/ncurses-6.1-nhodl3ckldkmn6nfzgtymzlsn7mm3htq/lib;/Users/tstockmanns/.spack/install_tree/darwin-mojave-x86_64/clang-10.0.1-apple/lz4-1.9.2-cnm2a54bm7elfkjfzhrpgnziyt75bzsv/lib;/Users/tstockmanns/.spack/install_tree/darwin-mojave-x86_64/clang-10.0.1-apple/openblas-0.3.7-glwve2w4uwiguptrtbygvyjkyfprfltm/lib;/opt/X11/lib;/opt/X11/lib;/Users/tstockmanns/.spack/install_tree/darwin-mojave-x86_64/clang-10.0.1-apple/pcre-8.42-3xya2nlccz7ryla4rtf54lwervn5kyh3/lib;/Users/tstockmanns/.spack/install_tree/darwin-mojave-x86_64/clang-10.0.1-apple/py-numpy-1.16.5-uoov6k5siu7qvx3h4asrzkfpo7m2ozxy/lib;/Users/tstockmanns/.spack/install_tree/darwin-mojave-x86_64/clang-10.0.1-apple/python-2.7.16-62pwxg2x3wcrcissw4g52hwyopv4htoh/lib;/Users/tstockmanns/.spack/install_tree/darwin-mojave-x86_64/clang-10.0.1-apple/expat-2.2.9-w2xd5p2t36nigz6raboofm5rrby7bbz6/lib;/Users/tstockmanns/.spack/install_tree/darwin-mojave-x86_64/clang-10.0.1-apple/gdbm-1.18.1-vt6s3h6mkxz3pzjlsobjoovxdfc2kvta/lib;/Users/tstockmanns/.spack/install_tree/darwin-mojave-x86_64/clang-10.0.1-apple/readline-8.0-vt4gbqchsmrmlrz6c7fggvc6dhiv2w34/lib;/Users/tstockmanns/.spack/install_tree/darwin-mojave-x86_64/clang-10.0.1-apple/libffi-3.2.1-rmioaofogfk7efvt3cpb3d433sevjnaf/lib;/Users/tstockmanns/.spack/install_tree/darwin-mojave-x86_64/clang-10.0.1-apple/sqlite-3.30.1-hysfl774rnexzt7l5bgfpboq6fs2mr4k/lib;/Users/tstockmanns/.spack/install_tree/darwin-mojave-x86_64/clang-10.0.1-apple/pythia6-428-alice1-3dsxc6ipbg7navctni6xt4psd7643hvz/lib;/Users/tstockmanns/.spack/install_tree/darwin-mojave-x86_64/clang-10.0.1-apple/pythia8-8240-hmzvl5d37vwyrh6s2mw7ybsjeiah25k6/lib;/Users/tstockmanns/.spack/install_tree/darwin-mojave-x86_64/clang-10.0.1-apple/hepmc-2.06.09-jlnuvva2u6uirf6kqf32diakyov3vlmr/lib;/Users/tstockmanns/.spack/install_tree/darwin-mojave-x86_64/clang-10.0.1-apple/unuran-1.8.1-6cvarkb7d6lq2wqpciyqan4e2ptctypf/lib;/Users/tstockmanns/.spack/install_tree/darwin-mojave-x86_64/clang-10.0.1-apple/rngstreams-1.0.1-z5cmigxer3zoo3rbfqk3y7wkkkdf5fhn/lib;/Users/tstockmanns/.spack/install_tree/darwin-mojave-x86_64/clang-10.0.1-apple/vc-1.4.1-muxusobwcmyil6hvsk2ri4clbnipimjp/lib;/Users/tstockmanns/.spack/install_tree/darwin-mojave-x86_64/clang-10.0.1-apple/xrootd-4.10.0-l5tyqlop6ehbitpbejpdahuo3mbhptw3/lib;/Users/tstockmanns/.spack/install_tree/darwin-mojave-x86_64/clang-10.0.1-apple/xxhash-0.6.5-6tsnapu2ppd6mcbn3xg467i7aez23bqc/lib;/Users/tstockmanns/.spack/install_tree/darwin-mojave-x86_64/clang-10.0.1-apple/geant4-10.05.p01-z7c3zt5v6bm4fa6av5payfkuazf27pu2/lib;/Users/tstockmanns/.spack/install_tree/darwin-mojave-x86_64/clang-10.0.1-apple/xerces-c-3.2.2-qbiitudag2akm7zouwzjwhwektjy2amy/lib;/Users/tstockmanns/.spack/install_tree/darwin-mojave-x86_64/clang-10.0.1-apple/geant4_vmc-4-0-p1-vzvptaevfos3hqbm6bxr7achm2wu5o7c/lib;/Users/tstockmanns/.spack/install_tree/darwin-mojave-x86_64/clang-10.0.1-apple/vgm-4-5-vmey35we5xin7xfjhn27ef77nqzx5xhl/lib;/Users/tstockmanns/.spack/install_tree/darwin-mojave-x86_64/clang-10.0.1-apple/protobuf-3.7.1-lc5hdspvfyoismlvb66szm6agganjal3/lib;/Users/tstockmanns/.spack/install_tree/darwin-mojave-x86_64/clang-10.0.1-apple/yaml-cpp-0.6.2-luo5siiqpwqi4wjncysazbacvwwlz2k4/lib' '-DCMAKE_PREFIX_PATH:STRING=/Users/tstockmanns/.spack/install_tree/darwin-mojave-x86_64/clang-10.0.1-apple/boost-1.68.0-3xkegvqksficzlnriyd7s4ikss2vp6hq;/Users/tstockmanns/.spack/install_tree/darwin-mojave-x86_64/clang-10.0.1-apple/cmake-3.16.2-zjrxq5csgjavghi4lijsmre5ghzhoygq;/Users/tstockmanns/.spack/install_tree/darwin-mojave-x86_64/clang-10.0.1-apple/fairlogger-1.4.0-4zg2pknqx2ixkof4cxdvmdyoyeurgzv6;/Users/tstockmanns/.spack/install_tree/darwin-mojave-x86_64/clang-10.0.1-apple/fairmq-1.4.3-hyprxzrxh33mjjc6fveohengs7jy5lnx;/Users/tstockmanns/.spack/install_tree/darwin-mojave-x86_64/clang-10.0.1-apple/flatbuffers-1.11.0-3z6x5fbnnnwrfg4kapi25jewahtrsnjp;/Users/tstockmanns/.spack/install_tree/darwin-mojave-x86_64/clang-10.0.1-apple/geant3-v2-7_fairsoft-yedc7jwoscuhn5eih72e3apnr26cye7v;/Users/tstockmanns/.spack/install_tree/darwin-mojave-x86_64/clang-10.0.1-apple/geant4-10.05.p01-z7c3zt5v6bm4fa6av5payfkuazf27pu2;/Users/tstockmanns/.spack/install_tree/darwin-mojave-x86_64/clang-10.0.1-apple/geant4_vmc-4-0-p1-vzvptaevfos3hqbm6bxr7achm2wu5o7c;/Users/tstockmanns/.spack/install_tree/darwin-mojave-x86_64/clang-10.0.1-apple/googletest-1.8.1-sdcowbfxteo6x2mnlzbuh3yobr6znwzf;/Users/tstockmanns/.spack/install_tree/darwin-mojave-x86_64/clang-10.0.1-apple/libxml2-2.9.9-rfcckf4qb62ko44mqt3xu2yg33h5o5qr;/Users/tstockmanns/.spack/install_tree/darwin-mojave-x86_64/clang-10.0.1-apple/protobuf-3.7.1-lc5hdspvfyoismlvb66szm6agganjal3;/Users/tstockmanns/.spack/install_tree/darwin-mojave-x86_64/clang-10.0.1-apple/pythia6-428-alice1-3dsxc6ipbg7navctni6xt4psd7643hvz;/Users/tstockmanns/.spack/install_tree/darwin-mojave-x86_64/clang-10.0.1-apple/pythia8-8240-hmzvl5d37vwyrh6s2mw7ybsjeiah25k6;/Users/tstockmanns/.spack/install_tree/darwin-mojave-x86_64/clang-10.0.1-apple/root-6.16.00-ejfra4res6lpxita2fjwqq5pzz3jzaxu;/Users/tstockmanns/.spack/install_tree/darwin-mojave-x86_64/clang-10.0.1-apple/vgm-4-5-vmey35we5xin7xfjhn27ef77nqzx5xhl;/Users/tstockmanns/.spack/install_tree/darwin-mojave-x86_64/clang-10.0.1-apple/yaml-cpp-0.6.2-luo5siiqpwqi4wjncysazbacvwwlz2k4' '-DROOTSYS=/Users/tstockmanns/.spack/install_tree/darwin-mojave-x86_64/clang-10.0.1-apple/root-6.16.00-ejfra4res6lpxita2fjwqq5pzz3jzaxu' '-DROOT_CONFIG_SEARCHPATH=/Users/tstockmanns/.spack/install_tree/darwin-mojave-x86_64/clang-10.0.1-apple/root-6.16.00-ejfra4res6lpxita2fjwqq5pzz3jzaxu' '-DPythia6_LIBRARY_DIR=/Users/tstockmanns/.spack/install_tree/darwin-mojave-x86_64/clang-10.0.1-apple/pythia6-428-alice1-3dsxc6ipbg7navctni6xt4psd7643hvz/lib' '-DPYTHIA8_DIR=/Users/tstockmanns/.spack/install_tree/darwin-mojave-x86_64/clang-10.0.1-apple/pythia8-8240-hmzvl5d37vwyrh6s2mw7ybsjeiah25k6' '-DGeant3_DIR=/Users/tstockmanns/.spack/install_tree/darwin-mojave-x86_64/clang-10.0.1-apple/geant3-v2-7_fairsoft-yedc7jwoscuhn5eih72e3apnr26cye7v' '-DGeant4_DIR=/Users/tstockmanns/.spack/install_tree/darwin-mojave-x86_64/clang-10.0.1-apple/geant4-10.05.p01-z7c3zt5v6bm4fa6av5payfkuazf27pu2' '-DBOOST_ROOT=/Users/tstockmanns/.spack/install_tree/darwin-mojave-x86_64/clang-10.0.1-apple/boost-1.68.0-3xkegvqksficzlnriyd7s4ikss2vp6hq' '-DBOOST_INCLUDEDIR=/Users/tstockmanns/.spack/install_tree/darwin-mojave-x86_64/clang-10.0.1-apple/boost-1.68.0-3xkegvqksficzlnriyd7s4ikss2vp6hq/include' '-DBOOST_LIBRARYDIR=/Users/tstockmanns/.spack/install_tree/darwin-mojave-x86_64/clang-10.0.1-apple/boost-1.68.0-3xkegvqksficzlnriyd7s4ikss2vp6hq/lib' '-DFlatbuffers_DIR=/Users/tstockmanns/.spack/install_tree/darwin-mojave-x86_64/clang-10.0.1-apple/flatbuffers-1.11.0-3z6x5fbnnnwrfg4kapi25jewahtrsnjp' '-DDISABLE_GO=ON' '-DBUILD_EXAMPLES:BOOL=ON' '-DFAIRROOT_MODULAR_BUILD=ON' '-DCMAKE_EXPORT_COMPILE_COMMANDS=ON' '-DBoost_NO_BOOST_CMAKE=ON'

2 errors found in build log:
     91       Argument not separated from preceding token by whitespace.
     92     Call Stack (most recent call first):
     93       cmake/modules/FairMacros.cmake:428 (find_package)
     94       CMakeLists.txt:180 (find_package2)
     95     This warning is for project developers.  Use -Wno-dev to suppress it.
     96
  >> 97     CMake Error at /Users/tstockmanns/PandaSoftware/FairSoft/dev/build/lib64/Geant4-10.5.0/Geant4Config.cmake:730:
     98       Parse error.  Function missing ending ")".  Instead found unterminated
     99       string with text ")
     100
     101        endif()
     102        unset(Geant4_FIND_REQUIRED_${_remaining})
     103

     ...

     112      ".
     113    Call Stack (most recent call first):
     114      cmake/modules/FairMacros.cmake:428 (find_package)
     115      CMakeLists.txt:180 (find_package2)
     116
     117
  >> 118    CMake Error at cmake/modules/FairMacros.cmake:428 (find_package):
     119      find_package Error reading CMake code from
     120      "/Users/tstockmanns/PandaSoftware/FairSoft/dev/build/lib64/Geant4-10.5.0/Geant4Config.cmake".
     121    Call Stack (most recent call first):
     122      CMakeLists.txt:180 (find_package2)
     123
     124
```

--- a/packages/fairroot/package.py
+++ b/packages/fairroot/package.py
@@ -69,9 +69,12 @@ class Fairroot(CMakePackage):
     patch('cmake_utf8.patch', when='@18.2.1')
     patch('fairlogger_incdir.patch', level=0, when='@18.2.1')
 
-    def setup_environment(self, spack_env, run_env):
+    def setup_build_environment(self, env):
+        super(Fairroot, self).setup_build_environment(env)
         stdversion=('-std=c++%s' % self.spec.variants['cxxstd'].value)
-        spack_env.append_flags('CXXFLAGS', '-std=c++%s' % self.spec.variants['cxxstd'].value)
+        env.append_flags('CXXFLAGS', '-std=c++%s' % self.spec.variants['cxxstd'].value)
+        env.unset('SIMPATH')
+        env.unset('FAIRSOFT_ROOT')
 
     def cmake_args(self):
         spec = self.spec

--- a/packages/r3broot/package.py
+++ b/packages/r3broot/package.py
@@ -42,4 +42,5 @@ class R3broot(CMakePackage):
     def setup_build_environment(self, env):
         env.set("FAIRROOTPATH", self.spec['fairroot'].prefix)
         env.set("SIMPATH", self.spec["fairroot"].prefix)
+        env.unset('FAIRSOFT_ROOT')
         env.append_path('CPATH', self.spec['fairlogger'].prefix.include)

--- a/test/buildsetup.sh
+++ b/test/buildsetup.sh
@@ -1,5 +1,8 @@
 #! /bin/bash
 
+export SIMPATH="$PWD/test/simpath_broken"
+export FAIRSOFT_ROOT="$SIMPATH"
+
 if [ -z "$LABEL" ]
 then
 	if which lsb_release >/dev/null

--- a/test/simpath_broken/bin/fairsoft-config
+++ b/test/simpath_broken/bin/fairsoft-config
@@ -1,0 +1,5 @@
+#! /bin/sh
+
+echo "fairsoft-config should not be used with spack" >&2
+
+exit 1

